hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/SparkMergeHelper.java

```diff
@@ -101,13 +101,13 @@ public void runMerge(HoodieTable<T, JavaRDD<HoodieRecord<T>>, JavaRDD<HoodieKey>
     } catch (Exception e) {
       throw new HoodieException(e);
     } finally {
+      if (null != wrapper) {
```

Comment:
Can you please also add a comment here to elaborating why this particular order is crucial

Reply:
Done.